navx-rotate-to-angle-arcade/physics.py

```diff
@@ -1,35 +1,34 @@
-
 from pyfrc.physics import motor_cfgs, tankmodel
 from pyfrc.physics.units import units
 
+
 class PhysicsEngine(object):
-  
     def __init__(self, physics_controller):
         '''
             :param physics_controller: `pyfrc.physics.core.PhysicsInterface` object
                                        to communicate simulation effects to
         '''
-        
+
         self.physics_controller = physics_controller
         self.position = 0
-        
+
         self.physics_controller.add_device_gyro_channel('navxmxp_i2c_1_angle')
         self.physics_controller.add_device_gyro_channel('navxmxp_spi_4_angle')
-        
+
         # Change these parameters to fit your robot!
-        bumper_width = 3.25*units.inch
-        
+        bumper_width = 3.25 * units.inch
+
         self.drivetrain = tankmodel.TankModel.theory(
-            motor_cfgs.MOTOR_CFG_CIM,           # motor configuration
-            110*units.lbs,                      # robot mass
-            10.71,                              # drivetrain gear ratio
-            2,                                  # motors per side
-            22*units.inch,                      # robot wheelbase
-            23*units.inch + bumper_width*2,     # robot width
-            32*units.inch + bumper_width*2,     # robot length
-            6*units.inch                        # wheel diameter
+            motor_cfgs.MOTOR_CFG_CIM,  # motor configuration
+            110 * units.lbs,  # robot mass
+            10.71,  # drivetrain gear ratio
+            2,  # motors per side
+            22 * units.inch,  # robot wheelbase
+            23 * units.inch + bumper_width * 2,  # robot width
+            32 * units.inch + bumper_width * 2,  # robot length
+            6 * units.inch,  # wheel diameter
```

Comment:
Hmm. Would using keyword arguments make this more readable?

Reply:
Oh, I see what you mean. Yeah, that might help.

Reply:
It's not really better. It's probably best to just disable formatting for that.